.github/workflows/snyk_sca_scan.yaml

```diff
@@ -27,6 +27,6 @@ jobs:
       - name: Perform SCA Scan
         continue-on-error: false
         run: |
-          snyk test --yarn-workspaces --strict-out-of-sync=false --detection-depth=6 --exclude=docker,Dockerfile --severity-threshold=critical
+          snyk test --debug --all-projects --strict-out-of-sync=false --detection-depth=6 --exclude=docker,Dockerfile --severity-threshold=critical
```

Comment:
What information is in the debug logs?

Reply:
@mschile Chris is on PTO for a couple of weeks. I'm unclear on the reasoning behind the inclusion of debug here, but after reviewing the delta in outputs, I don't think the additional log level is useful at the moment. I'll update the PR to drop `--debug`. Appreciate you pointing this out.